clang/test/CXX/drs/cwg6xx.cpp

```diff
@@ -83,6 +83,7 @@ namespace cwg603 { // cwg603: 3.1
   typedef S<'\001'> S1;
   typedef S<(1ul << __CHAR_BIT__) + 1> S1;
   // since-cxx11-error@-1 {{non-type template argument evaluates to 257, which cannot be narrowed to type 'unsigned char'}}
+  //   since-cxx11-note@-4 {{template parameter is declared here}}
```

Comment:
Use a label instead of `-4`, readers shouldn't have to count lines to understand which line the note is emitted for.